src/Settings/AdvancedParameters.as

```diff
@@ -24,6 +24,26 @@ namespace PluginSettings
         RMC_MX_Url = UI::InputText("MX Base URL", RMC_MX_Url);
         UI::SetPreviousTooltip("Use this URL for API calls to ManiaExchange. Useful for hosting your own service for caching and preloading API responses for better performance.\nOnly change if you know what you're doing!");
 
+#if TMNEXT
+        if (UI::Button("Use official TMX API")) {
+            RMC_MX_Url = "https://" + MX_URL;
+        }
+
+        bool clickedDanApi = UI::Button("Use DanOnTheMoon's Preloading + Proxy API");
+        UI::SetPreviousTooltip("Much faster API that preloads random maps and proxies requests to TMX.");
+        if (clickedDanApi) {
+            RMC_MX_Url = "https://mx.danonthemoon.dev/mx";
+        }
```

Comment:
this api has not gone public and doesnt really need a dedicated button atm

Reply:
Commented for the moment until it's ready (sorry, shouldn't have added in the first place but didn't want to make the PR just for my API)